packages/orbit-components/src/Accordion/components/AccordionWrapper/index.tsx

```diff
@@ -1,48 +1,15 @@
-import styled, { css } from "styled-components";
 import React from "react";
+import cx from "clsx";
 
 import type { Props } from "./types";
-import defaultTheme from "../../../defaultTheme";
-import type { Theme, ThemeProps } from "../../../defaultTheme";
-import mq from "../../../utils/mediaQuery";
 
-export const getBorder = ({ theme }: ThemeProps): string =>
-  `${theme.orbit.borderWidthCard} ${theme.orbit.borderStyleCard} ${theme.orbit.borderColorCard}`;
-
-export const getBorderRadius = ({ theme }: ThemeProps): string =>
-  `${theme.orbit.borderRadiusNormal}`;
-
-const CardElement = css`
-  ${({ theme, expanded }: { theme: Theme; expanded?: boolean }) => css`
-    width: 100%;
-    box-sizing: border-box;
-    position: relative;
-    box-shadow: ${expanded && theme.orbit.boxShadowActionActive};
-    border-top: ${!expanded && getBorder};
-    background: ${theme.orbit.backgroundCard};
-    ${mq.largeMobile(css`
-      border-left: ${!expanded && getBorder};
-      border-right: ${!expanded && getBorder};
-    `)};
-  `};
-`;
-
-const StyledAccordionWrapper = styled.div<{ expanded?: boolean }>`
-  ${CardElement};
-  border: ${getBorder};
-  border-radius: ${getBorderRadius};
-  transition: margin ${({ theme }) => theme.orbit.durationFast} ease-in-out;
-  margin: ${({ theme }) => theme.orbit.spaceXSmall} 0;
-`;
-
-StyledAccordionWrapper.defaultProps = {
-  theme: defaultTheme,
-};
-
-const AccordionWrapper = ({ dataTest, initialExpanded, ...props }: Props) => (
-  <StyledAccordionWrapper
+const AccordionWrapper = ({ dataTest, ...props }: Props) => (
+  <div
+    className={cx(
+      "border-card rounded-normal duration-fast my-xs bg-white-normal relative w-full border border-solid transition-[margin] ease-in-out",
```

Comment:
The background class should be `bg-elevation-flat`, as it is the referenced replacement for the deprecated `backgroundCard` token.

The same could be said about the border width class. The replacement token is `elevationFlatBorderSize`. However, I also prefer to keep the native `border` class. The above one makes sense because we are talking about colors, that can be mapped in a different theme. But, in this case, I don't think border widths should be configurable per theme, so I'd leave it as is.